NitroxModel/Logger/Log.cs

```diff
@@ -34,7 +34,7 @@ public static void InGame(string msg)
             Info(msg);
         }
 
-        private static void Write(string fmt, params object[] arg)
+        public static void Write(string fmt, params object[] arg)
```

Comment:
Sorry this was my fault... we don't want to use write.  We want to use Log.Info().  Info means information log level which is just general info for someone reading the logs.

Let's set this back to private.